### ISSUE TITLE
Magma Orb

### ISSUE THREAD

**Issue by PureWaterEx** [Openarl/PathOfBuilding#657]
[c42UTA9J.txt](https://github.com/Openarl/PathOfBuilding/files/1346440/c42UTA9J.txt)

**Comment by dein0s** [comment]
Wut?
Can you provide a bit more description on this issue?

**Comment by Openarl** [comment]
Yeah, I'd love to help, but you've literally given me nothing to work with.
